data/json/monsters/kraken_spawn.json

```diff
@@ -0,0 +1,148 @@
+[
+  {
+    "id": "mon_octupus_stalker",
+    "type": "MONSTER",
+    "name": { "str": "Stalker Octopus", "str_pl": "Stalker Octopi" },
+    "description": "This octopus seems larger than you remember from visits to the aquarium and doesn't seem as slowed by land as would be expected.  It's body seems to mimic it's surroundings.",
+    "default_faction": "kraken",
+    "bodytype": "human",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "KRAKEN" ],
+    "volume": "90 L",
+    "weight": "120 kg",
+    "hp": 100,
+    "speed": 70,
+    "material": [ "flesh" ],
+    "symbol": "C",
+    "color": "white_magenta",
+    "aggression": 6,
+    "morale": 60,
+    "melee_skill": 2,
+    "melee_dice": 3,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "stab", "amount": 4, "armor_penetration": 4 } ],
+    "dodge": 3,
+    "armor_bash": 2,
+    "armor_cut": 8,
+    "armor_bullet": 12,
+    "special_attacks": [ { "id": "barbed_tentacle" } ],
+    "vision_day": 30,
+    "vision_night": 7,
+    "path_settings": { "allow_open_doors": true, "avoid_traps": true, "avoid_sharp": true, "max_dist": 10 },
+    "anger_triggers": [ "HURT" ],
+    "harvest": "kraken",
+    "reproduction": { "baby_monster": "mon_octupus_stalker_spawn", "baby_count": 1, "baby_timer": 700 },
+    "baby_flags": [ "SPRING" ],
+    "flags": [ "SEES", "HEARS", "SMELLS", "SWIMS", "PATH_AVOID_DANGER_1", "NIGHT_INVISIBILITY", "WARM", "ATTACKMON" ]
```

Comment:
I think either CAMOUFLAGE or WATER_CAMOUFLAGE would work better than NIGHT_INVISIBILITY, and ATTACKMON is deprecated by the factions (and would lead to them hitting each other if they path through each other's tile).